### PR TITLE
Update scythemedia.json

### PR DESCRIPTION
**Pull request by InariFoxed**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by MaskDuck — COMMENTED** [review]
I want to see a preview of your website.

**Comment by ScytheMediaTheDev** [comment]
Ok, here it is.
Screenshot_20240430_064017_Samsung Internet

**Comment by ScytheMediaTheDev** [comment]
It looks better on desktop

**Comment by ScytheMediaTheDev** [comment]
Bc it's designed mainly for desktop

**Comment by ScytheMediaTheDev** [comment]
I will just send the code.

**Comment by ScytheMediaTheDev** [comment]
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>ScytheMedia Portfolio</title>
    <style>
        body {
            background-color: #333;
            /* Dark grey background */
            color: lime;
            font-family: 'Courier New', Courier, monospace;
            margin: 0;
            padding: 0;
        }

        .header {
            text-align: center;
            padding: 20px;
        }

        .code-2 {
            background-color: #1e1e1e;
            /* Black background for code block */
            color: yellow;
            /* yellow text color */
            padding: 20px;
            margin: 20px;
            border-radius: 20px;
            font-family: 'Courier New', Courier, monospace;
            white-space: pre-wrap;
            /* Preserves whitespace and line breaks */
        }

        .code-block {
            background-color: #1e1e1e;
            /* Black background for code block */
            color: cyan;
            /* Cyan text color */
            padding: 20px;
            margin: 20px;
            border-radius: 20px;
            font-family: 'Courier New', Courier, monospace;
            white-space: pre-wrap;
            /* Preserves whitespace and line breaks */
        }
    </style>
</head>

<body>
    <div class="header">
        <h1>Scythe Media's Info</h1>
    </div>
    <div class="code-2">
        <p>cd website
        
sudo nano index.html
--------------------------------------------
██████████████████████████████  100%
--------------------------------------------

Succesfully ran command.</p>
    <a class="code-block">
        <p>// Developer Profile
            <p>class.A-Developer {</p>
            <p> class= body.text {</p>
            <p> my.name = "ScytheMedia";</p> my.role = "Average Developer";</p> my.interests = ["coding", "gaming"];</p>
        <p>// Description</p>
        <p> An average developer with a passion for building innovative solutions and a keen interest in gaming. Always eager to learn new technologies and create engaging user experiences.
            <p>end.of.body-text</p>
    </div>
</body>

</html>

**Comment by ScytheMediaTheDev** [comment]
Looks trash on github markdown. L

File: domains/scythemedia.json [hunk 1]
```diff
@@ -3,7 +3,15 @@
...
 	},
-	"record": {
-		"MX": ["mx1.improvmx.com"] 
+	  "record": {
+    "A": [
+      "185.199.108.153"
+      "185.199.109.153"
+      "185.199.110.153"
+      "185.199.111.153"
+    ],
```
Comment (CuteDog5695) [inline domains/scythemedia.json:12]: you need to add a comma for the first IP, second IP and third IP

File: domains/scythemedia.json [hunk 2]
```diff
@@ -3,7 +3,15 @@
...
+    "MX": [
+	"mx.improvmx.com"
+		  ]
```
Comment (CuteDog5695) [inline domains/scythemedia.json:15]: i think you need to add mx1.improvmx.com and mx2.improvmx.com, then add a TXT for improvMX to work

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by ScytheMediaTheDev** [comment]
Thanks!

**Comment by ScytheMediaTheDev** [comment]
@CuteDog5695 @MaskDuck , I have made the correct changes that @CuteDog5695 has requested.

**Review by iostpa — APPROVED** [review]
i think it should be fine now

**Comment by CuteDog5695** [comment]
oh and i modified MX and added TXT so that it would work with ImprovMX :)

**Comment by ScytheMediaTheDev** [comment]
Thanks!

**Pull request merged**